README.md

```diff
@@ -46,14 +46,14 @@ qsb() {
 	file="$(
 		FZF_DEFAULT_COMMAND="$RG_PREFIX '$1'" \
 			fzf \
-			--preview="if [[ -n {} ]]; then if [[ -n {q} ]]; then ~/bin/bat-extras/src/batgrep.sh --color=always --terminal-width=105 --context=3 {q} {}; else bat --color=always {}; fi; fi" \
+			--preview="if [[ -n {} ]]; then if [[ -n {q} ]]; then /usr/bin/batgrep --color=always --terminal-width=105 --context=3 {q} {}; else bat --color=always {}; fi; fi" \
```

Comment:
Can you please add an example how to install batgrep so you can call it like this, `/usr/bin/batgrep`? I suppose it's simply  symlink, but I think it will still be useful and more user friendly since all other tools in this function have installation instructions.